src/datreant/core/collections.py

```diff
@@ -439,6 +439,17 @@ def relpaths(self):
         return [member.relpath for member in self]
 
     @property
+    def parents(self):
```

Comment:
I forget that the `View` is an aggregate because the name doesn't suggest it to me. Did you ever think about calling it a `Forest` instead?